lib/ansible/modules/network/netvisor/pn_vrouter_pim_config.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Pluribus Networks
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: pn_vrouter_pim_config
+author: "Pluribus Networks (@rajaspachipulusu17)"
+version_added: "2.8"
+short_description: CLI command to modify vrouter-pim-config
+description:
+  - This module can be used to modify pim parameters.
+options:
+  pn_cliswitch:
+    description:
+      - Target switch to run the CLI on.
+    required: false
+    type: str
+  state:
+    description:
+      - State the action to perform. Use C(update) to modify the vrouter-pim-config.
+    required: true
+    type: str
+    choices: ['update']
```

Comment:
@rajaspachipulusu17  if this module take cares of only updating, then is it required from the user to give the state information, or in future you are planning to add other state options as well.

Reply:
@justjais it supports `update` only, just that we wanted to maintain same standards in all the modules we are following choice option here.

Reply:
@rajaspachipulusu17 ok